app/javascript/controllers/autocomplete_controller.js

```diff
@@ -0,0 +1,44 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
```

Comment:
nitpick: On pourrait avoir plusieurs types d'autocomplete (par exemple un autocomplete qui va taper un endpoint de search lorsque l'on entre une valeur). Peut-être qu'on aimerait avoir un controller différent pour ces cas-là, et donc avoir un nom plus précis pour les différencier.

Reply:
Est-ce que tu crois que ça vaut la peine de faire cette distinction tant qu'on en a qu'un ? Si un autre se rajoute le changement ne sera pas couteux et en attendant c'est peut-être plus clair comme ça, qu'en penses-tu ?

Reply:
ok sounds good to me 👍